cmd/root/sweet.go

```diff
@@ -8,22 +8,21 @@ import (
 	"math/rand"
 	"os"
 	"path"
-
 	"strings"
 	"time"
 
 	"github.com/NicksPatties/sweet/cmd/about"
 	"github.com/NicksPatties/sweet/cmd/add"
 	"github.com/NicksPatties/sweet/cmd/stats"
 	"github.com/NicksPatties/sweet/cmd/version"
-	. "github.com/NicksPatties/sweet/constants"
-	. "github.com/NicksPatties/sweet/db"
-	. "github.com/NicksPatties/sweet/util"
-	"github.com/spf13/cobra"
+	c "github.com/NicksPatties/sweet/constants"
+	db "github.com/NicksPatties/sweet/db"
```

Comment:
I don't know why I can't just remove the `db` from this line and have everything be ok. Try that.